README.md

```diff
@@ -52,15 +60,17 @@ rot8 --touchscreen <TOUCHSCREEN>
 
 ```
 
-there are the following args.
+This will start the daemon running, continuously checking for rotations.
+
+There are the following args:
 
 ```
 
---sleep                 // Set sleep millis (500)
+--sleep                 // Set millis to sleep between rotation checks (500)
 --display               // Set Display Device (eDP-1)
 --touchscreen           // Set Touchscreen Device X11, allows multiple devices (ELAN0732:00 04F3:22E1)
---keyboard              // Set keyboard to deactivate upon rotation
---threshold             // Set a rotation threshold between 0 and 1 (0.5)
+--keyboard              // Set keyboard to deactivate upon rotation (Sway only)
```

Comment:
--keyboard is not  available with X just sway at the moment.

Reply:
Agreed. That is what my change intended to convey. I may misunderstand?

Reply:
thanks, merged...